site/gatsby-site/cypress/e2e/integration/report.cy.js

```diff
@@ -7,6 +7,24 @@ describe('Report pages', () => {
     cy.visit(url);
   });
 
+  it('Should Display associated Incidents', () => {
+    cy.visit(`/reports/1`);
+
+    cy.contains('Associated Incidents').should('be.visible');
+
+    cy.contains('Google’s YouTube Kids App Presents Inappropriate Content')
+      .should('be.visible')
+      .should('have.attr', 'href', '/cite/1/');
+
+    cy.get('[data-cy="incident-report-card"]').should('have.length', 1);
+  });
+
+  it('Should not display associated Incidents', () => {
+    cy.visit(`/reports/2974/`);
+
+    cy.contains('Associated Incidents').should('not.exist');
+  });
+
   it('Should always be expanded', () => {
     cy.visit(url);
```

Comment:
reading the test linearly is a little confusing because the `url` identifier is bound to a test constant above.

Could you use and re-use the same identifiers?

Maybe: `report_with_incidents` + `report_no_incidents` ?